frontend/src/app/components/wp-table/configuration-modal/tabs/highlighting-tab.component.html

```diff
@@ -12,7 +12,7 @@
   </div>
   <form>
     <p [textContent]="text.highlighting_mode.description"></p>
-    <div class="form--field -full-width">
+    <div class="form--field -full-width inline-attribute-select">
```

Comment:
the `inline-attribute-select` class is added only for testing purposes. Is no other selector possible?